### PR TITLE
C#: Add body to partial methods

### PR DESCRIPTION
**Pull request by tamasvajk**

### CODE REVIEW

File: csharp/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
```
Comment (tamasvajk) [inline csharp/.vscode/launch.json:1]: I'm not sure we want to push this. Another option would be to `.gitignore` this file.

Comment (calumgrant) [reply csharp/.vscode/launch.json:1]: I don't think we should push this - add to `.gitignore`

**Comment by hvitved** [comment]
> Can we have some tests for this?

I agree. You can add for instance the test mentioned here: https://github.com/github/codeql-csharp-team/issues/48#issuecomment-668536295.

**Comment by tamasvajk** [comment]
I though about adding the test, and then decided that the AST printing would already cover testing this too (through  `csharp/ql/test/library-tests/partial/PrintAst.qll`). I'll add a test anyways, that way this PR is more self contained.

**Comment by tamasvajk** [comment]
@calumgrant I think the `partial` modifier is already extracted. We are storing all source level modifiers.

**Review by hvitved — APPROVED** [review]

**Pull request merged**